app/src/test/java/space/taran/arknavigator/ExampleUnitTest.kt

```diff
@@ -1,8 +1,7 @@
-package space.taran.arknavigator
-
 import org.junit.Test
```

Comment:
I think we can remove this file for now.

Reply:
Thank you for your valuable comment; The requested change has been applied in the latest commit.